conda_smithy/configure_feedstock.py

```diff
@@ -859,6 +861,84 @@ def migrate_combined_spec(combined_spec, forge_dir, config, forge_config):
     return combined_spec
 
 
+def _conda_build_api_render_for_smithy(
+    recipe_path,
+    config=None,
+    variants=None,
+    permit_unsatisfiable_variants=True,
+    finalize=True,
+    bypass_env_check=False,
+    **kwargs,
+):
+    """This function works just like conda_build.api.render, but it returns all of metadata objects
+    regardless of whether they produce a unique package hash / name.
+
+    When conda-build renders a recipe, it returns the metadata for each unique file generated. If a key
+    we use at the top-level in a multi-output recipe does not explicitly impact one of the recipe outputs
+    (i.e., an output's recipe does use that key), then conda-build will not return all of the variants
```

Comment:
Just trying to understand, but shouldn't this be the other way around?

```suggestion
    (i.e., an output's recipe doesn't use that key), then conda-build will not return all of the variants
```

Reply:
Yes sorry. Thank you!